internal/app/rds/rds.go

```diff
@@ -309,7 +334,12 @@ func (r *RDSFetcher) computeInstanceMetrics(dbInstance aws_rds_types.DBInstance,
 }
 
 // getLogFilesSize returns the size of all logs on the specified instance
-func (r *RDSFetcher) getLogFilesSize(dbidentifier string) (*int64, error) {
+func (r *RDSFetcher) getLogFilesSize(ctx context.Context, dbidentifier string) (*int64, error) {
+	_, span := tracer.Start(ctx, "collect-instance-log")
+	defer span.End()
+
+	span.SetAttributes(attribute.String("db.instance.id", dbidentifier))
```

Comment:
I would recommend you to put all the attribute in a dedicated module with constant, a bit like in https://github.com/open-telemetry/opentelemetry-go/blob/v1.24.0/semconv/v1.24.0/trace.go.

You could even use directly the semconv package for some of them to make sure you use the standard attributes names (db for example: https://pkg.go.dev/go.opentelemetry.io/otel@v1.24.0/semconv/v1.24.0#DBInstanceID)

i'll be easier to maintain/share between modules in the future.

Reply:
Thanks for that great reference!

I moved this to `semcov` and other span attribute to a `trace` package in  https://github.com/qonto/prometheus-rds-exporter/pull/118/commits/cfd57d247a70bb2ac364013b99713df07a801a33

Reply:
As discussed, I implemented it.